src/bb-modules/Formbuilder/Service.php

```diff
@@ -344,12 +344,12 @@ public function formFieldNameExists($data)
         $sql = ("
         SELECT COUNT( * )
         FROM  `form_field`
-        WHERE form_id = :form_id
-        AND name =  :field_name
-        AND id != :field_id
+        WHERE form_id = form_id
+        AND name =  field_name
+        AND id != field_id
         ");
 
-        $result = $this->di['db']->findOne('FormField', 'form_id = ? and name = ? and id != ?', array($form_id, $field_name, $field_id));
+        $result = $this->di['db']->findOne('FormField', 'form_id = ? and name = ? and id != ?', array('$form_id, $field_name, $field_id'));
```

Comment:
Is conversion to string necessary?

Reply:
no.

Reply:
I ask the question because of the following error.
 Box\Mod\Formbuilder\ServiceTest::testduplicateForm
TypeError: Cannot access offset of type string on string

Reply:
This is wrong! It's an array of parameters and not a string. The previous one is the correct one.